exotic/exotic.py

```diff
@@ -2322,115 +2316,114 @@ def main():
                                     f"{round(100 * myfit.residuals.std() / np.median(myfit.data), 6)}%")
                             log.debug(f"The Mean Squared Error is: {round(np.sum(myfit.residuals ** 2), 6)}\n")
 
-                            resstd = myfit.residuals.std() / np.median(myfit.data)
-                        if minSTD > resstd and myfit is not None:  # If the standard deviation is less than the previous min
-                            bestCompStar = j + 1
-                            comp_coords = compStarList[j]
-                            minSTD = resstd
-                            minAperture = aper
-                            minAnnulus = annulus
-                            # arrayNormUnc = arrayNormUnc
+                            res_std = myfit.residuals.std() / np.median(myfit.data)
+                        if photometry_info['min_std'] > res_std and myfit is not None:  # If the standard deviation is less than the previous min
                             ref_flux_opt2 = True
 
-                            # sets the lists we want to print to correspond to the optimal aperature
-                            goodFluxes = np.copy(myfit.data)
-                            # goodNormUnc = np.copy(myfit.dataerr)
-                            nonBJDTimes = np.copy(myfit.time)
-                            # nonBJDPhases = np.copy(myfit.phase)
-                            goodAirmasses = np.copy(myfit.airmass)
-                            goodTargets = tFlux1
-                            goodReferences = cFlux1
-                            goodTUnc = tFlux1 ** 0.5
-                            goodRUnc = cFlux1 ** 0.5
-                            # goodResids = myfit.residuals
-                            bestlmfit = myfit
-
-                            finXTargCent = psf_data["target"][aper_mask][:, 0]
-                            finYTargCent = psf_data["target"][aper_mask][:, 1]
-                            finXRefCent = psf_data[ckey][aper_mask][:, 0]
-                            finYRefCent = psf_data[ckey][aper_mask][:, 1]
+                            photometry_info.update(best_fit_lc=copy.deepcopy(myfit),
```

Comment:
Why deepcopy here?

Reply:
Due to `myfit` being overwritten, it is an object of the `lc_fitter` class. If I saved it without it, it's a shallow copy of the object, and once I overwrite it, it may affect the reference of the `photometry_info.` I use `deepcopy` to avoid that situation since I have encountered it before, but it's been a bit, so I can remove it.

Reply:
Sounds good. Resolved.